### DISCUSSION TITLE
Core modules still load when -d flag is set and run with the synthea jar

### DISCUSSION THREAD

**Discussion by r0sies** [synthetichealth/synthea#1412] [Q&A] [answered]
Hi,
I am running synthea using the .jar file. I have added the flag -d to use a local directory (that is bound to a mounted directory). In the local directory that contains the modules, there is only one custom module (.json) file in the folder (it still has all the submodule folders and look up table). When synthea is executed, the core modules that come with synthea are still loaded, despite not being in the local directory. Is there a way to use only the modules in the local directory and not those contained in the jar? Thanks

**Comment by hadleynet** [comment, marked as answer]
You can use https://github.com/synthetichealth/synthea/wiki/The--M-Feature to accomplish this but, as noted on the page, this can have many side effects and is only really recommended for debugging custom modules.
